homeassistant/components/meteoclimatic/const.py

```diff
@@ -51,67 +53,80 @@
     "temp_current": {
         SENSOR_TYPE_NAME: "Temperature",
         SENSOR_TYPE_UNIT: TEMP_CELSIUS,
+        SENSOR_TYPE_ICON: None,
```

Comment:
`SENSOR_TYPES[sensor_type].get(SENSOR_TYPE_ICON)`

Meaning you don't have to set them explicitly, as `None` will be returned from `get()` when it is missing.
```suggestion
```

Reply:
Done.